src/components/common/YourCompany/YourCompany.vue

```diff
@@ -433,21 +423,16 @@ export default class YourCompany extends Mixins(
     return this.toDisplayPhone(this.nrApplicant.phoneNumber)
   }
 
-  /** The recognition date or business start date-time string. */
-  get recognitionDateTime (): string {
+  /** The recognition date or business start date string. */
+  protected get recognitionDate (): string {
     if (this.isBenIaCorrectionFiling) {
       if (this.getCorrectedFilingDate) {
-        return this.apiToPacificDateTime(this.getCorrectedFilingDate)
+        return this.yyyyMmDdToPacificDate(this.getCorrectedFilingDate) as string
```

Comment:
No. The method returns a JS Date object. If you just cast it as a string then it uses the browser locale's toString() function to format it.

Instead, look in the date mixin for an appropriate method(s).

Reply:
Will do